app/assets/stylesheets/publisher.scss

```diff
@@ -29,25 +29,37 @@
     }
 
     .options_and_submit {
-      padding: 10px 0 ;
-      #publisher_service_icons {
-        .btn-link{ text-decoration: none; }
-        .btn-link.question_mark .entypo-cog {
+      padding: 10px 0;
+      .btn{ height: 32px; }
```

Comment:
Well I'm sorry, I really don't see the difference. Except maybe only that the aspect button seemed a little higher than the others before :/